packages/omd/omd.2.0.0~alpha1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "A Markdown frontend in pure OCaml"
+description: """
+This Markdown library is implemented using only pure OCaml (including
+I/O operations provided by the standard OCaml compiler distribution).
+OMD is meant to be as faithful as possible to the original Markdown.
+Additionally, OMD implements a few Github markdown features, an
+extension mechanism, and some other features. Note that the opam
+package installs both the OMD library and the command line tool `omd`."""
+maintainer: ["Nicolás Ojeda Bär <n.oje.bar@gmail.com>"]
+authors: [
+  "Philippe Wang <philippe.wang@gmail.com>"
+  "Nicolás Ojeda Bär <n.oje.bar@gmail.com>"
+]
+license: "ISC"
+tags: ["org:ocamllabs" "org:mirage"]
+homepage: "https://github.com/ocaml/omd"
+bug-reports: "https://github.com/ocaml/omd/issues"
+depends: [
+  "dune" {>= "2.5"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ocaml/omd.git"
+x-commit-hash: "a01807ec69c03781db9149d8a6759db049c936b8"
+url {
+  src:
+    "https://github.com/ocaml/omd/releases/download/v2.0.0_TILDE_alpha1/omd-v2.0.0~alpha1.tbz"
```

Comment:
```suggestion
    "https://github.com/ocaml/omd/releases/download/2.0.0.alpha1/omd-2.0.0.alpha1.tbz"
```

looks like the released tarball does not have tildes in the name

Reply:
I had missed your comment, sorry

Reply:
I merged this in the meantime, so we can check if and where we will need to add upper bounds

Reply:
Mmh the checksums are still wrong...